nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -183,7 +183,13 @@ update /etc/fstab.
     # parted /dev/sda -- mkpart primary 1MB -8GB
     ```
 
-3.  Finally, add a *swap* partition. The size required will vary
+3.  Set the *root* partition's boot flag to on. This allows the disk to be booted from.
```

Comment:
It's the partition with the bootloader installed into it.

In this example, this is the only partition with a filesystem on it, so root and boot partitions are the same.

But putting the emphasis on the `root` partition is confusing, especially for people who are used to having a boot partition and root partition.

Reply:
That's fair enough, I'll remove the emphasis on a new commit.